javascript/javascript3/week3/homework-js3-week3/screenshotss.js

```diff
@@ -0,0 +1,141 @@
+const BASE_URL_SCREENSHOT_GENERATOR = 'https://website-screenshot6.p.rapidapi.com';
+const API_HOST_SCREENSHOT_GENERATOR = 'website-screenshot6.p.rapidapi.com';
+
+const BASE_URL_CRUD_API = `https://crudcrud.com/api/${API_KEY_CRUD_API}`;
+
+const input = document.getElementsByClassName('url-input')[0];
+const button = document.getElementsByClassName('get-screenshot-button')[0];
```

Comment:
In the same way you are specifying what the button on line 10 is doing, you could do the same with this variable name.